### PR TITLE
New announcement and added decipher to the team

### PR DESCRIPTION
**Pull request by jmcudd**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,14 +1,13 @@
 # Announcement
-
-### Jumpstate (along with jumpsuit) is now offically *DEPRECATED*. Thanks for your support over the life of this library. It served a great purpose, but we now are strongly urging you to use idiomatic (vanilla) redux in conjunction with redux-thunk. It will most certainly improve your apps testability, stability, and scalability.
+Jumpstate is now officially supported by [Decipher](http://deciphernow.com/) however Jumpsuit is still depreciated.
```
Comment (danielpcox) [inline README.md]: "deprecated", not "depreciated"

**Comment by jmcudd** [comment]
Fixed.

**Comment by danielpcox** [comment]
👍 Merging

**Comment by tannerlinsley** [comment]
Good job guys :) Great to see some live breathed into this!

**Comment by danielpcox** [comment]
Thanks! Our pleasure.

**Pull request merged**